CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## [1.0.2] - 2021-09-02
+
+### Added
+- Experimental support for custom authorization policies based on Open Policy Agent (OPA) (#2416)
+- SPIRE Server can now be configured to emit audit logs (#2297, #2391, #2394, #2396, #2442, #2458)
+- SDS v3 API in agent now supports the SPIFFE Certificate Validator (#2435, #2460)
```

Comment:
A little bit more about what we get, and that it is related to envoy, could be good here.

```suggestion
- Envoy SDS v3 API in agent now supports the SPIFFE Certificate Validator for federated SPIFFE authentication (#2435, #2460)
```